bin/deploy-prod.sh

```diff
@@ -9,6 +9,12 @@ set -o pipefail
 # echo out each line of the shell as it executes
 set -x
 
+# Update the circle envvars for all apps
+cf set-env gov-au-beta-blue CIRCLE_TAG "$CIRCLE_TAG"
+cf set-env gov-au-beta-blue CIRCLE_SHA1 "$CIRCLE_SHA1"
+cf set-env gov-au-beta-green CIRCLE_TAG "$CIRCLE_TAG"
+cf set-env gov-au-beta-green CIRCLE_SHA1 "$CIRCLE_SHA1"
+
```

Comment:
Any reason why you're setting these before rather than after the deploy has run?